aplib.net-demo/Assets/Scripts/Entities/Enemy/MeleeEnemy.cs

```diff
@@ -6,10 +6,11 @@
 /// Melee enemy that attacks the player when in range.
 /// It inherits from DummyEnemy to add respawn functionality.
 /// </summary>
+[RequireComponent(typeof(Animator))]
```

Comment:
Should this be here in the MeleeEnemy class? Or would this fit better within abstract enemy? Since we want both enemies to have animations later.